bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigToml.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.runtime.internal.configurable;
+
+import io.ballerina.runtime.internal.configurable.exceptions.TomlException;
+import io.ballerina.toml.semantic.ast.TomlTableNode;
+import io.ballerina.toml.semantic.ast.TomlTransformer;
+import io.ballerina.toml.syntax.tree.DocumentNode;
+import io.ballerina.toml.syntax.tree.SyntaxTree;
+import io.ballerina.tools.diagnostics.Diagnostic;
+import io.ballerina.tools.text.LineRange;
+import io.ballerina.tools.text.TextDocument;
+import io.ballerina.tools.text.TextDocuments;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.ballerina.runtime.internal.configurable.ConfigurableConstants.INVALID_TOML_FILE;
+
+/**
+ * Represents configuration TOML document for `configurable` variables.
+ *
+ * @since 2.0.0
+ */
+public class ConfigToml {
```

Comment:
As explained by Hinduja, this is a duplication of the following class.
https://github.com/ballerina-platform/ballerina-lang/blob/6aa21902043111c5776679afdf28fafd071c8aa2/compiler/ballerina-lang/src/main/java/io/ballerina/projects/TomlDocument.java#L36

@HindujaB Can you please create an issue to fix this? We need to bring this class to some other module.

Reply:
Created issue #27475